lib/LaTeXML/Package/fancyhdr.sty.ltxml

```diff
@@ -24,19 +24,19 @@ use LaTeXML::Package;
 
 ######################################################
 
-DefMacroI('\lhead', '{}', '');
-DefMacroI('\chead', '{}', '');
-DefMacroI('\rhead', '{}', '');
+DefMacroI('\lhead', '[]{}', '');
+DefMacroI('\chead', '[]{}', '');
+DefMacroI('\rhead', '[]{}', '');
 
-DefMacroI('\lfoot', '{}', '');
-DefMacroI('\cfoot', '{}', '');
-DefMacroI('\rfoot', '{}', '');
+DefMacroI('\lfoot', '[]{}', '');
+DefMacroI('\cfoot', '[]{}', '');
+DefMacroI('\rfoot', '[]{}', '');
 
 DefMacroI('\fancyhead', '[]{}', '');
 DefMacroI('\fancyfoot', '[]{}', '');
 
-DefMacroI('\fancypagestyle', '{}{}', '');
-DefMacroI('\fancyhf',        '[]',   '');
+DefMacroI('\fancypagestyle', '{}[]{}', '');
+DefMacroI('\fancyhf',        '[]{}',   '');
 
 DefMacroI('\fancyheadoffset', '[]{}', '');
 DefMacroI('\fancyfootoffset', '[]{}', '');
```

Comment:
Probably useful (upon reading the manual) to add **```DefMacro('\fancyhfoffset[]{}','');`` as well.

There are several other macros, but seem intended to be used inside the heads/foots, which we never expand, so presumably we can ignore them.

Reply:
I reordered the macros to match the docs because I kept getting lost. `\fancyhfoffset` was actually there, but I did miss another macro. Now I am fairly sure it's all there. Also, I fixed `\iffootnote`, because it's not a conditional.

*Except*, pdflatex does not compile the test, mostly because the test uses the wrong `\iffootnote` (presumably because of an old version?). I am converting the PR to draft because I need to figure out another couple of things first.